docs/reference/search/suggesters/completion-suggest.asciidoc

```diff
@@ -104,6 +104,10 @@ The following parameters are supported:
     which defines a weight and allows you to rank your suggestions.
     This field is optional.
 
+NOTE: The `input` value must not contain any of the three reserved UTF-16
+control characters `Null (\u0000)`, `Information Separator Two (\u001e)` or
+`Information Separator One (\u001f)`
```

Comment:
@jrodewig I'm not too sure if this is too technical at this point in the docs since its unlikely that most users will have these characters in their input, but it came up in an issue most likely during testing and I though we at least should note the reserved characters somewhere. Happy to move it anywhere else, would like to hear your thoughts.

Reply:
Thanks @cbuescher. I don't believe it's too technical, but I would move the note so it directly follows the `input` parm definition:

```
`input`::
  The input to store, this can be an array of strings or just
  a string. This field is mandatory.
+
NOTE: This value cannot contain the following UTF-16 control characters:

* `\u0000` (null)
* `\u001f` (information separator one)
* `\u001e` (information separator two)
```

Note I added a `+` here so it is indented with the rest of the `input` definition.